src/js/legend.js

```diff
@@ -0,0 +1,222 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("gpii.chartAuthoring.pieChart.legend", {
+        gradeNames: ["gpii.d3ViewComponent", "autoInit"],
+        strings: {
+            legendColHeading:"Legend",
+            labelColHeading:"Label",
+            valueColHeading:"Value"
+        },
+        model: {
```

Comment:
Sorry I didn't make it clear. What I meant of adding a comment for "dataSetWithColors" was,

```javascript
model: {
     dataSet: [],
    // "dataSetWithColors" consolidates the dataset in original order with the array of colors, into a single array of objects
    // dataSetWithColors: []
}
```
When writing component API docs, we normally look at this defaults block. This comment helps to realize there's another model path that's created internally.